dist/org.duckstation.DuckStation.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.duckstation.DuckStation</id>
+  <launchable type="desktop-id">org.duckstation.DuckStation.desktop</launchable>
```

Comment:
Side question: do we want duckstation-nogui or duckstation-qt to be the default application (org.duckstation.DuckStation.desktop)? The nogui is quite strange because it has - in fact - a GUI.

Personally I prefer nogui, even though I'm on KDE.

Reply:
Yes. NoGUI is mainly intended for TV-like environments, and doesn't have all the options.